common/djangoapps/util/tests/test_password_policy_validators.py

```diff
@@ -0,0 +1,31 @@
+"""Tests for util.password_policy_validators module."""
+
+import unittest
+
+from django.core.exceptions import ValidationError
+from django.test.utils import override_settings
+
+from util.password_policy_validators import validate_password_dictionary
+
+
+class PasswordPolicyValidatorsTestCase(unittest.TestCase):
```

Comment:
We have tests for this already in common/djangoapps/students/, but those go through a POST layer and are thus slightly more integrate-y than these tests I've added. I wanted to throw in a few unit tests closer to the source.